fairscale/nn/wrap/auto_wrap.py

```diff
@@ -13,18 +13,20 @@ def default_auto_wrap_policy(
     module: nn.Module,
     recurse: bool,
     unwrapped_params: int,
+    module_is_root: bool,
     # These are customizable for this default policy function.
     min_num_params: int = int(1e8),
     force_leaf_modules: Optional[Set[Type[nn.Module]]] = None,
     exclude_wrap_modules: Optional[Set[Type[nn.Module]]] = None,
+    skip_params_check_for_root: bool = True,
```

Comment:
Won't this change behavior is users are currently using this policy? Wondering if users will end up doing a double wrap without knowing we've already wrapped the root module.

Reply:
good catch. It is worth checking. If so, perhaps default it the False? Also, I can make a separate change where we disable double wrapping in FSDP's init function.

Reply:
Makes sense. Let's make the default behavior as  False for now and once we have the PR that disables double wrapping in FSDP's init function, we can make the default behavior as being True. I was thinking if there might be any repercussions for nested wrapping etc. but I think not wrapping if the module has already been wrapped is a fair check in any condition.